doc/changes/latest.inc

```diff
@@ -12,15 +12,15 @@
 Current (0.20.dev0)
 -------------------
 
-- Add reader for NIRx data in :func:`mne.io.read_raw_nirx` by `Robert Luke`_
```

Comment:
bad rebase

Reply:
nope.  I moved it because it was in the wrong spot.